package-lock.json

```diff
@@ -11727,6 +11721,12 @@
         "symbol-observable": "1.2.0"
       }
     },
+    "redux-offline": {
+      "version": "git+https://github.com/enahum/redux-offline.git#4bd85e7e3b279a2b11fb4d587808d583d2b5e7b5",
+      "requires": {
+        "redux-persist": "4.10.2"
+      }
+    },
```

Comment:
I don't really know why is moving it here.

Reply:
is probably re-sorting them, no need to worry